FordLabs/retroquest
# Commonalize configuration properties

## Overview
Commonalize Configuration Properties

See #408 for full context

### Notes
This pull request does not move all properties to `@ConfigurationProperties` classes. 

## Testing Instructions
1. Build and test the application locally
2. Ensure everything continues to work as expected
